ra2ce/graph/network_config_data/network_config_data_validator.py

```diff
@@ -78,7 +78,7 @@ def _wrong_value(self, key: str) -> str:
 
     def _wrong_enum(self, key: str, enum: Ra2ceEnumBase) -> str:
         # Remove last value INVALID
-        _accepted_values = enum.list_valid_options()
+        _accepted_values = type(enum).list_valid_options()
```

Comment:
`type` here should not be needed. If so, there's something wrong in the code.
https://docs.python.org/3/library/functions.html#classmethod